packages/manager/modules/core/src/index.js

```diff
@@ -254,6 +254,14 @@ export const registerCoreModule = (environment) => {
         $urlServiceProvider.config.strictMode(false);
       },
     )
+    .config(
+      /* @ngInject */ ($logProvider) => {
+        $logProvider.debugEnabled(
+          !WEBPACK_ENV.production ||
```

Comment:
I'm a bit confused to have a webpack variable in a module buit with rollup

Reply:
That's a good point!

Right now, I do not have a clear way to handle this case.
So I'll close this Pull Request and re-open it once I will have a proper solution to suggest.

Thanks for the review!